backend/app/api/endpoints/adapter/shells.py

```
@@ -0,0 +1,741 @@
+    # Soft delete
+    shell.is_active = False
+    db.add(shell)
+    db.commit()
+
+    return {"message": "Shell deleted successfully"}
+
+
+@router.post("/validate-image", response_model=ImageValidationResponse)
+async def validate_image(
+    request: ImageValidationRequest,
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.6)

Unused function argument: `current_user`

(ARG001)